core/src/test/java/org/apache/calcite/test/TableFunctionTest.java

```diff
@@ -359,16 +359,10 @@ private Connection getConnectionWithMultiplyFunction() throws SQLException {
   }
 
   @Test public void testUserDefinedTableFunction4() {
-    final String q = "select *\n"
+    final String q = "select \"c1\"\n"
```

Comment:
Why we also change the query content ?

Reply:
Because now type coercion is supported,  running this test will get a valid result.
So I change this test to make it the same like `testUserDefinedTableFunction3`.